app.js

```diff
@@ -35,6 +35,44 @@ async function hashText(message) {
   return hash384;
 }
 
+function parseContentType(res) {
```

Comment:
nit: The function should get the header value as a string, you should do the "res.headers.get()" in the caller of this function.